### ISSUE TITLE
Hindi Text Layer Selection Some part of the text is extracted wrong

### ISSUE THREAD

**Issue by viju2008** [mozilla/pdf.js#12554]
Link of the document 

https://github.com/mozilla/pdf.js/files/3460804/mahageeta1.pdf

Configuration:

Web browser and its version: Google Chrome
Version 86.0.4240.111 (Official Build) (64-bit)
Operating system and its version: Windows 10
PDF.js version: pdf.js 2.6.401

Steps to reproduce the problem:

Load the above PDF with PDF.js 

Try to highlight the Hindi Text it is rendered and seen properly
But the Text layer is showing wrong text 

The Document is visible as below
https://user-images.githubusercontent.com/1583316/62364487-38adeb80-b53f-11e9-8bc2-6ceb05b2500e.png

On highlighting the Text the following is coming in Text Layer 

https://user-images.githubusercontent.com/1583316/62364507-46637100-b53f-11e9-9d8a-b82fd548ddd0.png

The Wrong hindi is displayed in Text layer

Please help 

Vijay

**Comment by viju2008** [comment]
Is there any font folder in pdf.js library where i can add new fonts so that this issue dosent occur

**Comment by Snuffleupagus** [comment]
First of all, both the PDF file and the screen-shots included here are not unique to this issue but rather appears to be taken *directly* from issue #11037; hence this looks like a duplicate of that old issue.

As clearly mentioned in https://github.com/mozilla/pdf.js/issues/11037#issuecomment-517662006, the textLayer isn't suppose to be rendered.

---

Secondly, and more importantly here, this is a bug in the PDF document *itself*, rather than in the PDF.js library, since the fonts in question contain *incomplete* `/ToUnicode` data and text-selection/copying thus cannot work. Please note that this is broken in other PDF viewer as well, e.g. Adobe Reader and PDFium/Google Chrome.

Basically, without the PDF document *itself* containing the necessary data there's nothing that can be done here; hence this issue is unfortunately INVALID.
